tracks/perl5/two-fer/mentoring.md

```diff
@@ -0,0 +1,111 @@
+# Mentoring
+
+This is a great opportunity to prepare for the track with attention to style conventions.
+This is currently the first reviewed problem on the Perl track, and serves to
+make sure the student understands unit testing and Test Driven Design.
+
+## Reasonable Solutions
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+sub two_fer {
+    my $name = shift || "you";
+    return "One for $name, one for me.";
+}
+1;
+```
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+sub two_fer {
+    return "One for " . (shift || "you") . ", one for me.";
+}
+1;
+```
+
+```perl
+package TwoFer;
+use strict;
+use warnings;
+use Exporter 'import';
+our @EXPORT_OK = qw(two_fer);
+use experimental qw(signatures);
+sub two_fer ($name = 'you') {
+    return "One for $name, one for me.";
+}
+1;
+```
+But remember TMTOWDI - There's More Than One Way To Do It!
+
+## Common Suggestions
+
+- Suggest using a default value instead of any form of conditionals.
+- If the student is using Perl 5.2 or newer, suggest using the `experimentals` module but make them aware that they're unlikely to find that particular pattern in the wild.
```

Comment:
```suggestion
- If the student is using Perl 5.2 or newer, suggest using the `experimentals` module but make them aware that they are unlikely to find that particular pattern in the wild.
```

Reply:
If this is early in the track, are we suggesting using `experimentals` module actively?  Should it be that we don't mention it until later in the track, or if they use it?  If it is not commonly used, then it seems like an "intermediate" style comment to make, otherwise.